ElectronClient/checkForUpdates.js

```diff
@@ -149,8 +149,10 @@ function checkForUpdates(inBackground, window, logFilePath, options) {
 
 			const result = await dialog.showMessageBox(parentWindow_, {
 				type: 'info',
-				message: `${_('An update is available, do you want to download it now?')}\n\n${_('Your version: %s', packageInfo.version)}\n${_('New version: %s', newVersionString)}${releaseNotes}`,
```

Comment:
Please restore the string "An update is available, do you want to download it now?"